monitoring/mongodb/mongo_metrics_with_compatible_mode.example

```diff
@@ -0,0 +1,4859 @@
+# HELP collector_scrape_time_ms Time taken for scrape by collector
```

Comment:
should we store this doc here?

Reply:
It will help us figure out dashboard/alert issues if any occur.